docs/UserGuide.md

```diff
@@ -221,12 +223,15 @@ editclass 1 [cn/CLASS_NAME] [ct/CLASS_TIMING] [r/RATE] [l/LOCATION]
 ```
 
 * Edits the class at the specified `INDEX`. The index refers to the index number shown in the displayed class
-  list in the `classes` tab. The index **must be a positive integer** 1, 2, 3, …​
+  list in the `classes` tab. 
+  * The index **must be a positive integer** 1, 2, 3, …​
+  * The index must also not be out of rage.
 * At least one of the optional fields must be provided.
 * `CLASS_TIMING` can only start at the hour mark or half hour mark.
+* Edit commands that will create a **clash of class timing** is not accepted.
```

Comment:
maybe can specify as clash of timing with other existing classes?

Reply:
ok done.